### PR TITLE
changelog for v1.2.3

### PR DESCRIPTION
**Pull request by mattcary**
/kind documentation

```release-note
None
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.2.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Update debian images to buster-v1.9.0 ([#839](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/839), [@mattcary](https://github.com/mattcary))
+
+### Uncategorized
+
+- It is now possible to mount a volume with XFS filesystem and its restored snapshot. ([#838](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/838), [@leiyiz](https://github.com/leiyiz))
```
Comment (jingxu97) [inline CHANGELOG/CHANGELOG-1.2.md:11]: maybe mention mount at the same time?

Comment (mattcary) [reply CHANGELOG/CHANGELOG-1.2.md:11]: Discussed offline. Decided not to mess with the auto-generated changelog.

**Comment by mattcary** [comment]
Failures are flakes or known old version problems.

**Pull request merged**